devices/camera.py

```diff
@@ -1109,6 +1017,25 @@ def expose_command(
             self.pre_foc = []
             self.pre_ocn = []
 
+            # Within each count - which is a single requested exposure, IF it is a smartstack
+            # Then we divide each count up into individual smartstack exposures.
+            ssExp=self.config["camera"][self.name]["settings"]['smart_stack_exposure_time']
+            if self.current_filter in ['HA', 'O3', 'S2', 'N2', 'y', 'up', 'U']:
+                ssExp = ssExp * 3 # For narrowband and low throughput filters, increase base exposure time.
+            if not imtype.lower() in ["light"]:
+                print ("skipping smartstack as not a lightframe")
+                Nsmartstack=1
+                SmartStackID='no'
```

Comment:
Just so you know, for variable naming it's not standard to use CamelCase/mixedCase styles in Python, we use the CapNames for classes and exceptions. The existing variables named this way are just from old code, so instead we should be naming our variables lowercase_with_underscores. I know this is a minor point but I felt it was worth mentioning. There are some other small formatting details here with spacing, which a linter can pick up and fix.

Reply:
It is good to be learning! My brain is a mess of multiple programming languages over time and different standards, so a lot of my stuff gets hybridised in my head. Variables with underscores, got it (but I will forget)